content/id/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -0,0 +1,351 @@
+---
+title: Cloud Providers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini akan menjelaskan bagaimana cara mengelola Kubernetes yang berjalan pada penyedia layanan cloud tertentu.
+{{% /capture %}}
+
+
+{{% capture body %}}
+### kubeadm
+[kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/) merupakan salah satu cara yang banyak digunakan untuk membuat kluster Kubernetes.
+kubeadm memiliki beragam opsi untuk mengatur konfigurasi spesifik untuk penyedia layanan cloud. Salah satu contoh yang biasa digunakan pada penyedia cloud *in-tree* yang dapat diatur dengan kubeadm sebagai berikut:
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: InitConfiguration
+nodeRegistration:
+  kubeletExtraArgs:
+    cloud-provider: "openstack"
+    cloud-config: "/etc/kubernetes/cloud.conf"
+---
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: ClusterConfiguration
+kubernetesVersion: v1.13.0
+apiServer:
+  extraArgs:
+    cloud-provider: "openstack"
+    cloud-config: "/etc/kubernetes/cloud.conf"
+  extraVolumes:
+  - name: cloud
+    hostPath: "/etc/kubernetes/cloud.conf"
+    mountPath: "/etc/kubernetes/cloud.conf"
+controllerManager:
+  extraArgs:
+    cloud-provider: "openstack"
+    cloud-config: "/etc/kubernetes/cloud.conf"
+  extraVolumes:
+  - name: cloud
+    hostPath: "/etc/kubernetes/cloud.conf"
+    mountPath: "/etc/kubernetes/cloud.conf"
+```
+
+Penyedia layanan cloud *in-tree* biasanya membutuhkan `--cloud-provider` dan `--cloud-config` untuk ditentukan sebelumnya pada *command lines* untuk [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) dan
+[kubelet](/docs/admin/kubelet/). Konten dari file yang ditentukan pada `--cloud-config` untuk setiap provider akan dijabarkan di bawah ini.
+
+Untuk semua penyedia layanan cloud eksternal, silakan ikuti instruksi pada repositori masing-masing penyedia layanan.
+
+## AWS
+Bagian ini akan menjelaskan semua konfigurasi yang dapat diatur saat menjalankan Kubernetes pada Amazon Web Services.
+
+### Nama Node
+
+Penyedia layanan cloud AWS menggunakan nama DNS private dari instance AWS sebagai nama dari obyek Kubernetes Node.
+
+### Load Balancers
+Anda dapat mengatur [load balancers eksternal](/docs/tasks/access-application-cluster/create-external-load-balancer/) sehingga dapat menggunakan fitur khusus AWS dengan mengatur anotasi seperti di bawah ini.
```

Comment:
nit: prefer singular instead of plural (remove 's')

also italic load balancer, menjadi: ... [*loadbalancer* eksternal] ...